### PR TITLE
Travis CI initial configuration + LICENSE + README edit

### PR DESCRIPTION
**Pull request by Renelvon**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -0,0 +1,7 @@
+mock>=1.0.1
+nose>=1.3.3
+pep8>=1.5.6
```
Comment (dionyziz) [inline requirements.txt:3]: Why is `pep8` a build requirement?

File: README.md [hunk 2]
```diff
@@ -1,3 +1,12 @@
+# llama 
+
+[![Build Status](https://travis-ci.org/Renelvon/llama.svg?branch=travis)](https://travis-ci.org/Renelvon/llama)
```
Comment (dionyziz) [inline README.md]: We should use the master branch on the upstream (my) repository here. Let's change it and then merge.

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+language: python
+python:
+  - "3.3"
+  - "3.2"
+# command to install dependencies
+install:
+  - "pip install -r requirements.txt"
```
Comment (dionyziz) [inline .travis.yml:7]: We can drop the quotes on this line and on the version lines above for readability.

File: requirements.txt (hunk 1 above)
Comment (Renelvon) [inline requirements.txt:3]: It is not strictly a build requirement; only `make beauty` requires it. I prefer to keep all package dependencies together.

File: .travis.yml [hunk 4]
```diff
@@ -0,0 +1,9 @@
...
+# command to run tests
+script: make test
```
Comment (dionyziz) [inline .travis.yml:9]: I think this should be merged after we have merged the "test" branch. "make test" should run unit tests and return 0 on success or 1 on failure, but right now it just runs ptest.sh, which isn't concerned about return values.

File: .travis.yml (hunk 3 above)
Comment (Renelvon) [inline .travis.yml:7]: OK. Will do.

File: README.md (hunk 2 above)
Comment (Renelvon) [inline README.md]: OK.

File: .travis.yml (hunk 4 above)
Comment (Renelvon) [inline .travis.yml:9]: That is not a problem of `ptest.sh`. We should make the `main` module terminate with an error on a bad program. Then, some bash equality test is enough.

File: requirements.txt (hunk 1 above)
Comment (dionyziz) [inline requirements.txt:3]: OK

File: .travis.yml (hunk 3 above)
Comment (dionyziz) [inline .travis.yml:7]: OK waiting for this change and will merge.

File: .travis.yml (hunk 4 above)
Comment (dionyziz) [inline .travis.yml:9]: Modified version looks good.

**Pull request merged**